classes/patterns/class-deepdive.php

```diff
@@ -58,12 +58,13 @@ public static function get_column_template(): string {
 	 * @param array $params Optional array of parameters for the config.
 	 */
 	public static function get_config( $params = [] ): array {
+		$classname = self::get_classname();
 		return [
 			'title'      => __( 'Deep Dive', 'planet4-blocks-backend' ),
 			'categories' => [ 'planet4' ],
 			'content'    => '
-				<!-- wp:group {"className":"block","align":"full","backgroundColor":"grey-05"} -->
-					<div class="wp-block-group alignfull block has-grey-05-background-color has-background">
+				<!-- wp:group {"className":"block ' . $classname . '","align":"full","backgroundColor":"grey-05"} -->
```

Comment:
What do you think if we use directly `self::get_classname()` instead of defining a new variable immutable class name value?

Reply:
I'm using a variable to store result so that I don't call the same function multiple times. But maybe it would be better to manually declare that classname as a static property or constant for each pattern class ?